.github/workflows/release-packages.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish
+on:
+  # NOTE: avoid using the manual execution unless is the only way to fix some issue,
+  # rather retry failed jobs in case of flakes. The manual execution can potentially
+  # cause the tags to point to a different commit that the one used to publish
+  # the packages
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+        startsWith(github.event.head_commit.message, 'chore(release): bump package versions')
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        # don't checkout a detatched HEAD
+        ref: ${{ github.head_ref }}
+
+        # this is important so git log has the whole history
+        fetch-depth: '0'
+
+    - name: Setup git
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
```

Comment:
This workflow should probably use the github app credentials

Reply:
does that mean just using the Create Github App Token action similar to the bump? would that be sufficient

Reply:
Yeah, that should be enough to setup the git credentials and configure the user email/name appropriately.